### PR TITLE
Publish package

### PR DESCRIPTION
**Pull request by wilwade**
Updates the export interface some and packs up for npm

## Description
- Interfaces. There are now two main ones along with all the others. In theory these two are the only ones you need most of the time.
- Packaged up for npm
- Lowercase amplica-labs for npm org

## TODO in a separate PR
- [ ] CI Publish on version to npm

## Testing
- `pnpm run pack:siwf`
- Then you can use `./packages/siwf/dist/amplica-labs-siwf-0.1.0.tgz` in another repo with just `npm i ...`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -185,22 +185,45 @@ An empty payload means that a user has closed the popup window before completing
 {}
 ```
 
+### Response Parameters
+
+One of two objects will have data: `signUp` or `signIn`.
```
[the remark this answers is not in the document]
Comment (JoeCap08055) [reply README.md:190]: Discussion for future--but for now let's make the doc match the current state of the code.

[the remark this answers is not in the document]
Comment (wilwade) [reply README.md:190]: Actually I believe I did update the code to match this. I'll tag it in the code.

File: packages/siwf/src/signin.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+export type ValidSignIn = {
+  msaId: string;
+  publicKey: string;
+};
+
+/**
+ * Validates the signin and throws an error if the validation fails.
+ */
+export async function validateSignin(
```
Comment (enddynayn) [inline packages/siwf/src/signin.ts:20]: How did we miss this one! Thanks Will!

File: packages/siwf/src/signin.ts [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    return await validateSignature(hexAddress, siwsPayload.signature, siwsPayload.message);
+  } catch (_e) {
+    return false;
+  }
+}
+
+/**
+ * Returns the MSA Id if valid
+ */
+export async function getMsaId(api: ApiPromise, siwsMessage: SiwsMessage): Promise<string | null> {
```
Comment (enddynayn) [inline packages/siwf/src/signin.ts:93]: Not a blocker and we can come back to this. I think a different name like `getMsaIdBySiws`. I am not sure if that is better but I think it avoid a naming collision.

**Review by enddynayn — APPROVED** [review]
LGTM amazing 👍

**Review by JoeCap08055 — APPROVED** [review]

File: packages/siwf/src/signin.ts (hunk 3 above)
Comment (wilwade) [reply packages/siwf/src/signin.ts:93]: Hmm... Yeah. I wanted a `isValid...`, but i also wanted the data. So this is what came out.

I wasn't going to export all these things, but the siwf example needed it to be able to do the checklist.

**Pull request merged**